requests_kerberos/kerberos_.py

```diff
@@ -348,13 +348,16 @@ def authenticate_server(self, response):
         Returns True on success, False on failure.
         """
 
+        response_token = _negotiate_value(response)
         log.debug("authenticate_server(): Authenticate header: {0}".format(
-            _negotiate_value(response)))
+            base64.b64encode(response_token).decode()
+            if response_token
+            else ""))
```

Comment:
Note that `pylint` and other tools will complain about non-lazy logger usage here.

You might want to do

```
log.debug("authenticate_server(): Authenticate header: %s", base64.b64encode(response_token).decode() if response_token else "")
```

With this approach if the debug level isn't high enough your code will avoid one string formatting operation.

To make it even lazier, you can do

```
if log.isEnabledFor(logging.DEBUG):
    log.debug(...)
```

Reply:
Happy to accept a PR if you wish to make the change